tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -435,6 +471,7 @@ avg[union(sym2range[:coefficients]...)]
 With a bit of work (this will be much easier in the future), we can also visualize the approximate marginals of the different variables, similar to `plot(chain)`:
 
 ```julia
+using Plots, StatsPlots
```

Comment:
A couple of things here:
- `StatsPlots` is meant to be a drop-in for `Plots`, i.e. we shouldn't have to import _both_ but only importing `StatsPlots` should suffice.
- It shouldn't be necessary to import `StatsPlots` (or `Plots`) in every block we want to use their functionality. Once should be enough.

Reply:
> A couple of things here:

Yes totally agree with these points. 

> * `StatsPlots` is meant to be a drop-in for `Plots`, i.e. we shouldn't have to import _both_ but only importing `StatsPlots` should suffice.
I think in the original [version](https://github.com/TuringLang/TuringTutorials/blob/master/tutorials/09-variational-inference/09_variational-inference.jmd), both `Plots` and `StatsPlots` are imported, so I just leave as it is.
As pointed out, `StatsPlots` is a drop-in replacement for `Plots`, I just removed Plots to test it - will follow this up once buildkite is working.

> * It shouldn't be necessary to import `StatsPlots` (or `Plots`) in every block we want to use their functionality. Once should be enough.
Yes, this is again the same issue with the previous buildkite failure - Initially I just imported them once, it cannot find the `Plots` module without importing it in every julia code block. 
I am removing the redundant imports and see how it works again - will follow this up.

Reply:
>  Initially I just imported them once, it cannot find the Plots module without importing it in every julia code block.

Yeah so I get that this is the issue, but my point is that this seems to be a somewhat deeper issue with how we use buildkite or something indicating that each block is not evaluated in the same module? It seems like something we should fix rather than hack around, since otherwise we'll have to do this "import in every block" for every tutorial going forwards :confused:

Reply:
Yes definitely need to look into this critical issue why the build environment doesn't make all imports store in the workspace - it seems each time it finishes executing a julia code block, it does some garbage collection thing? 
However, it just happens to the module `Plots`. Is it because it has got two choices, i.e. `Plots` and `StatsPlots`? and automatically clear up `Plots`? Maybe some dependency settings?

Reply:
Sorry, just saw this comment. And yeah, it's probably because both `Plots` and `StatsPlots` are `export` ing the same functions, e.g. `histogram`, and so Julia requires you to use the namespace when calling the function. Hence if you just do `using StatsPlots` once and remove all `using Plots` statements, the issue should go away:)

Reply:
Thanks very much for the thoughts - yes I removed all `Plots` imports and only kept `StatsPlots`, it solves the problem.